clients/imodels-client-authoring/src/base/internal/LocalFileSystemImpl.ts

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { constants, promises } from "fs";
+
+import { LocalFileSystem } from "../public";
+
+export class LocalFileSystemImpl implements LocalFileSystem {
```

Comment:
Rather than using a generic name with abbreviation, you could call it NodeLocalFileSystem for example.